agwa-section1-accounts-domains-directory.md

```
@@ -0,0 +1,1324 @@
+```
+
+| 設定項目 | 推奨値 | 説明 |
+| --------- | -------- | ------ |
+| 最低文字数 | 12文字 | 短いパスワードは総当たり攻撃に弱い |
+| 最大文字数 | 100文字 | パスフレーズを許可する |
+| 強度の強制 | 有効 | 大文字・小文字・数字・記号を含む |
+| 期限切れ | 設定しないことを推奨 | NIST のガイドラインに準拠 |
+| 再利用禁止 | 有効（最近のパスワードを禁止） | 古いパスワードへの回帰を防ぐ |
+
+> ⚠️ **試験ポイント**: NIST の最新ガイドライン（SP 800-63B）では、定期的なパスワード変更の強制は**推奨されない**場合があります。侵害の証拠がある場合のみ変更を要求するアプローチが現在の主流です。
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)